crates/router/src/connector/stripe/transformers.rs

```diff
@@ -843,131 +848,81 @@ fn validate_shipping_address_against_payment_method(
 }
 
 fn infer_stripe_pay_later_type(
-    pm_type: &enums::PaymentMethodType,
+    pay_later_data: &api_models::payments::PayLaterData,
     experience: &enums::PaymentExperience,
```

Comment:
There is no need to pass experience to this or any method since it is being used only in payment_experience and that also will be removed